bundles/org.dataflowanalysis.analysis.dfd/src/org/dataflowanalysis/analysis/dfd/interactive/DFDAnalysisCLI.java

```diff
@@ -0,0 +1,124 @@
+package org.dataflowanalysis.analysis.dfd.interactive;
+
+import org.apache.log4j.Logger;
+import org.dataflowanalysis.analysis.core.FlowGraphCollection;
+import org.dataflowanalysis.analysis.dfd.DFDConfidentialityAnalysis;
+import org.dataflowanalysis.analysis.dfd.DFDDataFlowAnalysisBuilder;
+import org.dataflowanalysis.analysis.dsl.AnalysisConstraint;
+import org.dataflowanalysis.analysis.dsl.result.DSLResult;
+import org.dataflowanalysis.analysis.utils.StringView;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class DFDAnalysisCLI {
+    private static final Logger logger = Logger.getLogger(DFDAnalysisCLI.class);
+
+    public static void main(String[] args) {
```

Comment:
JavaDoc that this is a potential entry point to the analysis that can be used for use case x